client/dashboard/style.scss

```diff
@@ -179,3 +179,44 @@
 		/* stylelint-enable */
 	}
 }
+
+.components-modal__frame.woocommerce-cart-modal {
```

Comment:
There a few minor differences in heading and p tag sizes between this and the design.

#### Current
<img width="604" alt="Screen Shot 2019-12-19 at 2 15 32 PM" src="https://user-images.githubusercontent.com/10561050/71149831-be220600-226a-11ea-9530-acd5c50958e1.png">


#### Design
<img width="575" alt="Screen Shot 2019-12-19 at 2 21 14 PM" src="https://user-images.githubusercontent.com/10561050/71149869-d5f98a00-226a-11ea-8283-b1a2ede17270.png">

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/3444/commits/d7762abe8ad6717ef3db01792f1b1fe9f3584906.